notification/app/services/Configuration.scala

```diff
@@ -0,0 +1,34 @@
+package services
+
+import javax.inject.Inject
+
+import com.gu.conf.ConfigurationFactory
+
+import scala.concurrent.ExecutionContext
+import scalaz.\/
+import scalaz.std.option.optionSyntax._
+
+case class NotificationHubConfiguration(
+  connectionString: String,
+  hubName: String
+)
+
+case class ErrorMessage(message: String)
+
+final class Configuration @Inject()()(implicit executionContext: ExecutionContext) {
+
+  private lazy val conf = ConfigurationFactory.getConfiguration(
+    applicationName = "notification",
+    webappConfDirectory = "gu-conf"
+  )
+
+  private def getConfigurationProperty(name: String): ErrorMessage \/ String =
```

Comment:
I doubt any part of the application should work if it's not configured properly, should we just throw an exception here instead of returning Either and letting every client throw their own exception?